app/values/file_info.rb

```diff
@@ -0,0 +1,24 @@
+class FileInfo
+  attr_reader :file_info
+  delegate :[], :has_key?, to: :file_info
+  def initialize(file_info)
+    @file_info = file_info
+  end
+
+  def to_h
+    file_info
+  end
+
+  def file_name
+    file_info["file_name"]
+  end
+
+  def file_path
+    FilePath.new(file_info["url"])
+  end
+
+  def mime_type
```

Comment:
I just checked and it looks like dropbox [does provide it](https://github.com/projecthydra-labs/browse-everything/blob/master/spec/fixtures/vcr_cassettes/dropbox.yml), so it would be nice to support that.